src/containers/ConnectorManagement.jsx

```diff
@@ -105,11 +107,25 @@ export default class ConnectorManagement extends Component {
   }
 
   gotoParent () {
+    this.clearLoginFormInputs()
+
     const router = this.context.router
     let url = router.location.pathname
     router.push(url.substring(0, url.lastIndexOf('/')))
   }
 
+  /*
+   * Empies Login Form values to prevent the browser (aka firefox) to propose to save the credentials
+   */
+  clearLoginFormInputs() {
+    const formElements = document.querySelectorAll(`.${stylesLoginForm['account-form-login']} input`)
```

Comment:
The React/preact philosophy is to avoid querying/updating the DOM in other way than using jsx in `render()` method.
So what we should do here instead should be to update our component state (to trigger à re-render) and pass a property to children component, until `AccountLoginForm` is re-rendered and use this property to render itself without values in input fields.